src/components/NetworkPanel/CyjsRenderer/CyjsRenderer.tsx

```diff
@@ -218,11 +218,47 @@ const CyjsRenderer = ({
       visualEditorProperties,
     )
 
-    // Generate a new Cytoscape.js styles based on given visual style
-    const newStyle = createCyjsDataMapper(vs)
-    setCyStyle(newStyle)
 
-    // Restore selection state in Cyjs instance
+    const newStyle = createCyjsDataMapper(vs);
```

Comment:
This code should belong in cyjs-util.ts, compute-view-util.ts, DataMapper.ts and DirectMappingSleector.ts